filesystem/src/test/java/org/frankframework/filesystem/sftp/SftpFileSystemTest.java

```diff
@@ -21,6 +22,7 @@
  *
  */
 @Log4j2
+@Tag("slow")
```

Comment:
Als alle filesystem tests langzaam zijn kan je deze beter op de parent class zetten, of op `BasicFileSystemTest` de `@Tag("slow")` en `@Tag("filesystem")`.

Reply:
Ik dacht alleen de LocalFileSystemTests nog uit te voeren in de standaard set voor de "Smoketest" build per JDK, dus daarom lijkt het me niet handig het op de parent-class te zetten.